### PR TITLE
exercise-12

### PR DESCRIPTION
**Pull request by vntrp**

### CODE REVIEW

File: exercise-12.txt [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
++-----------+----------+
+|         5 |       19 |
+|        14 |       19 |
+|        58 |       18 |
++-----------+----------+
+3 rows in set (0.00 sec)
+
+mysql> 
+
+// Answer: 2 accounts own the most Address Books: accounts 5 & 14.
```
Comment (nyanofthemoon) [inline exercise-12.txt]: :+1:

File: exercise-12.txt [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
++---------------+----------+
+|            37 |       52 |
+|            20 |       50 |
+|            50 |       49 |
++---------------+----------+
+3 rows in set (0.00 sec)
+
+mysql> 
+
+// Answer: Address Book 37 contains the most entries.
```
Comment (nyanofthemoon) [inline exercise-12.txt]: :+1:

**Comment by nyanofthemoon** [comment]
Nice for going out of your way to get more than just the top row :)

**Comment by vntrp** [comment]
haha! sure thing, it was a good remark from your part!

**Pull request closed without merging**